### DISCUSSION TITLE
Introduction

### DISCUSSION THREAD

**Discussion by yashwanth-yasp** [community/community#19977] [Other Feature Feedback, Questions, & Ideas]
Hello everybody
I'm Yashwanth A Doddegowda, and I'm enrolled in the Indian Institute of Information Technology to study computer science. I enjoy programming and am currently studying ui/ux design. I am somewhat new to the field of computer science but I have a lot of passion towards it, thus I am open to any and all advice that might help me get better.

**Comment by vietdoo** [comment]
Hope you have fun learning and practicing more new skills.

**Comment by manasesjesus** [comment]
Hello and welcome to GH, Yashwanth. You may consider creating a profile readme to introduce yourself. It's fairly simple and there are different ways to accomplish it.

Create a repo named as your username (**yashwanth-yasp**) and add a README. The info in your readme will be displayed on your profile ✨automagically✨. You can use [this tool](https://rahuldkjain.github.io/gh-profile-readme-generator/) to generate the contents of your readme, and [this one](https://github.com/anuraghazra/github-readme-stats) to add your statistics. Here's GitHub's step-by-step [profile README tutorial](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme).

Another approach is using GitHub Actions to update your readme whenever an event triggers the action. There are actions such as [this](https://github.com/marketplace/actions/github-update-readme), [this](https://github.com/marketplace/actions/update-action-readme), and [many more](https://github.com/marketplace?type=actions&query=readme+). You can add all the info you want to make it handsome.

You can also code your own action using GitHub GraphQL and REST APIs to get the info you want, and update the readme with that info. It can be displayed in SVG and you can use CSS for colors and animations. Take a look at [my profile](https://github.com/manasesjesus) to see this approach.

Finally, take a look at [this repository](https://github.com/abhisheknaiidu/awesome-github-profile-readme) as it has a collection of readme profiles. You may get inspiration by looking at different profiles.
